src/flagsView.ts

```diff
@@ -0,0 +1,452 @@
+import * as vscode from 'vscode';
+import { FeatureFlag } from './models';
+import { LaunchDarklyAPI } from './api';
+import { Configuration, getIsTreeviewEnabled } from './configuration';
+import { FlagStore } from './flagStore';
+import * as path from 'path';
+
+export class ldFeatureFlagsProvider implements vscode.TreeDataProvider<FlagValue> {
```

Comment:
Since this is a pretty large provider, I agree we should move it to a new file. It might make sense now to create a new subdirectory called `providers` and toss them all in there.

nit: for consistency with other providers, I'd recommend calling this `LaunchDarklyTreeViewProvider`